### PR TITLE
Add community asking units / category

### PR DESCRIPTION
**Pull request by VonOx**
Adding

* Data category + units
* Datarate category + units
* Units for precipitations
* 
https://community.gladysassistant.com/t/nouvelle-feature-flux-reseau/6609
https://community.gladysassistant.com/t/test-avant-release-v4-10-0-broadlink-volets-roulants-performance-lqi-z2m-voc-z2m-keep-history-webcals/7527/5?u=vonox

### CODE REVIEW

File: front/src/config/i18n/en.json [hunk 1]
```diff
@@ -2037,6 +2073,14 @@
       "state": "Curtain state",
       "position": "curtain position"
     },
+    "data": {
+      "shortCategoryName": "Datas",
```
Comment (Pierre-Gilles) [inline front/src/config/i18n/en.json]: Data is always singular!

File: front/src/config/i18n/fr.json [hunk 2]
```diff
@@ -1743,7 +1744,24 @@
...
+    "gigabyte": "Gigabyte (GB)",
+    "terabyte": "Terabyte (TB)",
+    "bits-per-second": "Bits par seconde (bit/s)",
+    "kilobits-per-second": "Kilobits par seconde (kbit/s)",
+    "megabits-per-second": "Megabits par seconde (Mbit/s)",
+    "gigabits-per-second": "Gigabits par seconde (Gbit/s)",
+    "bytes-per-second": "Bytes par seconde (B/s)",
+    "kilobytes-per-second": "kilobytes par seconde (kB/s)",
+    "megabytes-per-second": "Megabytes par seconde (MB/s)",
+    "gigabytes-per-second": "Gigabytes par seconde (GB/s)"
```
Comment (Pierre-Gilles) [inline front/src/config/i18n/fr.json]: en français on parle de gigaoctets

File: front/src/config/i18n/fr.json [hunk 3]
```diff
@@ -1795,7 +1814,24 @@
...
+    "gigabyte": "GB",
+    "terabyte": "TB",
+    "bits-per-second": "bit/s",
+    "kilobits-per-second": "kbit/s",
+    "megabits-per-second": "Mbit/s",
+    "gigabits-per-second": "Gbit/s",
+    "bytes-per-second": "B/s",
+    "kilobytes-per-second": "kB/s",
+    "megabytes-per-second": "MB/s",
+    "gigabytes-per-second": "GB/s"
```
Comment (Pierre-Gilles) [inline front/src/config/i18n/fr.json]: Go en français

**Comment by Pierre-Gilles** [comment]
@VonOx Seems this PR is failing?

**Comment by VonOx** [comment]
Codecov upload....

**Review by Pierre-Gilles — APPROVED** [review]

**Pull request merged**